src/interface.c

```diff
@@ -1279,22 +1280,73 @@ static void draw_processes(struct device_info *dev_info,
   print_processes_on_screen(total_processes, &interface->process);
 }
 
-static const char *signalsName[] = {
-    "Cancel",  "SIGABRT", "SIGALRM",   "SIGBUS",  "SIGCHLD", "SIGCONT",
-    "SIGFPE",  "SIGHUP",  "SIGILL",    "SIGINT",  "SIGKILL", "SIGPIPE",
-    "SIGQUIT", "SIGSEGV", "SIGSTOP",   "SIGTERM", "SIGTSTP", "SIGTTIN",
-    "SIGTTOU", "SIGUSR1", "SIGUSR2",   "SIGPOLL", "SIGPROF", "SIGSYS",
-    "SIGTRAP", "SIGURG",  "SIGVTALRM", "SIGXCPU", "SIGXFSZ",
+static const char *signalNames[] = {
+  "SIGHUP",
```

Comment:
Keep `"Cancel"` as the fist option please.
The code below assumes that the first of the list is cancel.

Reply:
Okay, but can I define `"Cancel"` as `SIGKILL` ?

Reply:
Done